src/current/_includes/molt/replicator-flags.md

```diff
@@ -0,0 +1,85 @@
+| `--gracePeriod`              | `DURATION` | Allow background processes to exit.<br><br>**Default:** `30s`                                                                                 |
+| `--logDestination`           | `STRING`   | Write logs to a file. If not specified, write logs to `stdout`.                                                                               |
+| `--logFormat`                | `STRING`   | Choose log output format: `"fluent"`, `"text"`.<br><br>**Default:** `"text"`                                                                  |
+| `--metricsAddr`              | `STRING`   | A `host:port` on which to serve metrics and diagnostics.                                                                                      |
+| `--parallelism`              | `INT`      | The number of concurrent database transactions to use.<br><br>**Default:** `16`                                                               |
+| `--quiescentPeriod`          | `DURATION` | How often to retry deferred mutations.<br><br>**Default:** `10s`                                                                              |
+| `--retireOffset`             | `DURATION` | How long to delay removal of applied mutations.<br><br>**Default:** `24h0m0s`                                                                 |
+| `--scanSize`                 | `INT`      | The number of rows to retrieve from staging.<br><br>**Default:** `10000`                                                                      |
+| `--schemaRefresh`            | `DURATION` | How often a watcher will refresh its schema. If this value is zero or negative, refresh behavior will be disabled.<br><br>**Default:** `1m0s` |
+| `--sourceConn`               | `STRING`   | The source database's connection string.                                                                                                      |
+| `--stageMarkAppliedLimit`    | `INT`      | Limit the number of mutations to be marked applied in a single statement.<br><br>**Default:** `100000`                                        |
+| `--stageSanityCheckPeriod`   | `DURATION` | How often to validate staging table apply order (`-1` to disable).<br><br>**Default:** `10m0s`                                                |
+| `--stageSanityCheckWindow`   | `DURATION` | How far back to look when validating staging table apply order.<br><br>**Default:** `1h0m0s`                                                  |
+| `--stageUnappliedPeriod`     | `DURATION` | How often to report the number of unapplied mutations in staging tables (`-1` to disable).<br><br>**Default:** `1m0s`                         |
+| `--stagingConn`              | `STRING`   | The staging database's connection string.                                                                                                     |
```

Comment:
Again, I was not sure what "staging database" referred to, until I read down the list to `--stagingSchema`.